### ISSUE TITLE
Labelling of SVGs

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#657]
This is a typical `<svg>` icon that appears multiple times on a search result page.
```
<svg viewBox="0 0 32 32" aria-labelledby="aria-label-stanford" role="image" aria-labelled-by="bl-icon-stanford-ad5f6a514493c764">
  <title>
    stanford
  </title>
  <path d="M5.77 5.01L10.2.6h11.7l4.32 4.42v6.25h-7.72V8.33h-4.68v4.23h8.47l3.93 3.8v11.5l-3.86 3.85H9.65l-3.88-3.9V21.6h8.34v2.57h4.75v-3.94H9.63l-3.86-3.86V5zm.5.23v10.93l3.56 3.56h9.53v4.97h-5.78v-2.57h-7.3v5.48l3.57 3.6h12.3l3.55-3.55V16.59l-3.64-3.5H13.3V7.85H19v2.91h6.7v-5.5l-4.03-4.13H10.4zm4.34-3.57h10.87l3.71 3.83v4.78h-5.66V7.36h-6.75v6.29h9.07l3.34 3.19v10.64l-3.25 3.26H10.06l-3.27-3.31v-4.75h6.28v2.57h6.8v-6h-9.83l-3.26-3.26V5.5z"></path>
<title id="bl-icon-stanford-ad5f6a514493c764">stanford icon</title></svg>
```

Issues:

0. the icons are never perceived by the screenreader because of #634 
1. element has two `aria-labelledby` attributes
   - aria-labelledby="aria-label-stanford"
   - aria-labelled-by="bl-icon-stanford-ad5f6a514493c764"
2. `aria-labelled-by` isn't a valid attribute.
3. only one of the attributes has a matching ID
   - `title id="bl-icon-stanford-ad5f6a514493c764"`
4. but that ID is repeated each time the icon appears, which violates ARIA rules
5. there are two `<title>` attributes
5. `<title>` with the ID (which would override the unassigned one for the screenreader) is "stanford icon", which is not a meaningful text equivalent

Can simplify this?
- remove the IDs and `aria-labelledby`
- use a single title, remove "icon", and use appropriate case for the text, since it also appears as the hover text/tooltip

_Source: https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html "SVGs that convey information"_

```
<svg viewBox="0 0 32 32" role="img" focusable="false">
  <path d="M5.77 5.01L10.2.6h11.7l4.32 4.42v6.25h-7.72V8.33h-4.68v4.23h8.47l3.93 3.8v11.5l-3.86 3.85H9.65l-3.88-3.9V21.6h8.34v2.57h4.75v-3.94H9.63l-3.86-3.86V5zm.5.23v10.93l3.56 3.56h9.53v4.97h-5.78v-2.57h-7.3v5.48l3.57 3.6h12.3l3.55-3.55V16.59l-3.64-3.5H13.3V7.85H19v2.91h6.7v-5.5l-4.03-4.13H10.4zm4.34-3.57h10.87l3.71 3.83v4.78h-5.66V7.36h-6.75v6.29h9.07l3.34 3.19v10.64l-3.25 3.26H10.06l-3.27-3.31v-4.75h6.28v2.57h6.8v-6h-9.83l-3.26-3.26V5.5z"></path>
<title>Stanford</title></svg>
```

This will both "read" as give a tooltip as "Stanford" and generates no violations.

I suspect there may be upstream issues that affect what we do with this, but we need to at a minimum remove the extra / bad attributes, make sure it's perceivable, and has a reasonable title/tooltip.

**Comment by mejackreed** [comment]
Some background here, GeoBlacklight went with an icon solution in place before Blacklight had theirs in place. So both are being used, thus the duplication. See https://github.com/geoblacklight/geoblacklight/pull/840

We should probably back part of that out of GeoBlacklight and use the Blacklight based solution.

**Comment by jvine** [comment]
Just noting this also applies to title icons in show page.
https://earthworks.stanford.edu/catalog/princeton-w3763818m

**Comment by mejackreed** [comment]
@jvine previously in Blacklight we had decided to use the "Semantic icon needs" upstream in Blacklight for ArcLight and other things.See https://github.com/projectblacklight/blacklight/issues/2150  The recommendation here removes some of that upstream. I'm just confirming that we have decided a different approach. This is further confused by the doubling of aria-labels because GeoBlacklight did its own thing which I think we should remove.

**Comment by mejackreed** [comment]
Maybe we are now moving from Pattern 5 -> Pattern 8 here: https://www.deque.com/blog/creating-accessible-svgs/

And I guess that is fine then?

**Comment by jvine** [comment]
Per convo with Jack:
keep title
add aria-label with same text as title
kill aria-labelledby
